schema/url/standalone-map/plugin-collection-markers.raml

```diff
@@ -0,0 +1,68 @@
+#%RAML 1.0 Library
+
+types:
+
+  StandaloneCollectionMapMarkerPostRequest:
+    type: DataPluginRequestBase
+    properties:
+      config: StandaloneCollectionMapMarkerSpec
+
+  StandaloneCollectionMapMarkerSpec:
+    additionalProperties: false
+    description: Specification for a map marker
+    properties:
+      outputEntityId:
+        required: true
+        type: string
+        description: Entity ID associated with collection variable of interest.
+      geoAggregateVariable:
+        type: VariableSpec
+        description: Geohash component variable to group markers by. This is usually determined by zoom level of the client application.
+      longitudeVariable:
+        type: VariableSpec
+        description: Variable representing longitude of output entity.
+      latitudeVariable:
+        type: VariableSpec
+        description: Variable representing latitude of output entity.
+      viewport:
+        type: GeolocationViewport
+        description: Viewport used to filter markers. Only entities that are contained in the viewport will be grouped into markers.
+      collection:
+        required: true
+        type: CollectionSpec
+        description: Collection variable to use for markers.
+      aggregationConfig:
+        required: true
+        type: QuantitativeOverlayConfig
+        description: Specification for how variable values will be aggregated into quantiative value shown in bar plot marker.
+      selectedMemberVariables:
+        required: true
+        type: VariableSpec[]
+        description: Variables in the provided collection for which marker data should be provided.
+
+  CollectionMapMarkerElement:
+    description: |
+      Data needed to render a collection map marker. Contains geographical data as well as per-variable collection var
+      aggregates needed to render bar plots with error bar.
+    properties:
+      entityCount: number
+      avgLat: number
+      avgLon: number
+      minLat: number
+      minLon: number
+      maxLat: number
+      maxLon: number
+      values: CollectionMemberAggregate[]
+
+  CollectionMemberAggregate:
+    properties:
+      variableId: string
+      mean: number
+      median: number
+      confidenceInterval: NumberRange
```

Comment:
This is great. But I think we should also include the count of rows contributing to each bars/ variables summary value and confidence interval. Could call it `n` maybe, don't remember what we called it for lineplot..